tests/phpunit/tests/customize/nav-menus.php

```diff
@@ -953,11 +953,6 @@ function test_filter_wp_nav_menu() {
 			'items_wrap'  => '<ul id="%1$s" class="%2$s">%3$s</ul>',
 		);
 
-		// Add global namespace prefix to check #41488.
-		if ( version_compare( PHP_VERSION, '5.3', '>=' ) ) {
-			$original_args['fallback_cb'] = '\\' . $original_args['fallback_cb'];
```

Comment:
I think this check needs to be inverted?  I.e. we should always run this line, rather than remove it.

Reply:
Yes - this is running a test to verify that the bug at https://core.trac.wordpress.org/ticket/41488 is fixed, but the test can only be run in PHP >= 5.4.

Reply:
To invert it, is it better to change `5.3` with `5.6` or should `>=` become `<=`?

Reply:
Better to remove the `if` block entirely and leave the code that was once inside it.

Reply:
done with latest commit